manifest.json

```diff
@@ -24,9 +24,9 @@
     "background": {
         "scripts": [
             "background/main.js",
+            "background/adapter/deluge.js",
             "background/adapter/rutorrent.js",
             "background/adapter/transmission.js"
         ]
     }
 }
-
```

Comment:
Please do not remove trailing new lines. Every file should end with am empty line.